codec-http2/src/test/java/io/netty/handler/codec/http2/Http2TestUtil.java

```diff
@@ -687,6 +687,14 @@ static ByteBuf bb(String s) {
         return ByteBufUtil.writeUtf8(UnpooledByteBufAllocator.DEFAULT, s);
     }
 
+    static ByteBuf bb(int size) {
+        ByteBuf buf = UnpooledByteBufAllocator.DEFAULT.buffer();
+        for (int i = 0; i < size; i++) {
+            buf.writeByte('a');
+        }
+        return buf;
+    }
```

Comment:
This feels like something that almost certainly exists somewhere else.

Reply:
Just use bytebuf.writeZero(size); ?

Reply:
Would you prefer I replace the body of this method with
`UnpooledByteBufAllocator.DEFAULT.buffer().writeZero(size)` or inline it in the best?

Reply:
I dont care... I was just mention it as you said there is most likely something like this already :)